package.json

```diff
@@ -19,6 +19,7 @@
     "test-cypress": "cypress open",
     "test-cypress-build": "cypress run --spec ./cypress/features/build/build*.feature --browser chrome --reporter spec",
     "test-cypress-accessibility": "cypress run --spec ./cypress/features/accessibility/accessibility.feature --browser chrome --reporter spec",
+    "test-cypress-visual": "cypress run --spec './cypress/features/visual/*.feature' --browser chrome --reporter spec",
```

Comment:
here we have the configuration for `--browser chrome` but in the setup we are using `Edge`

Reply:
The test runs in chrome but then we use applitools to test edge, dosnt matter anyway as it has been decided to only test chrome